auto_py_to_exe/web/js/i18n.js

```diff
@@ -294,6 +324,7 @@ const translationMap = {
                 de: 'VERZEICHNIS',
                 pt_br: 'DIRETÓRIO',
                 sp: 'Directorio',
+                tr: '.ico Dosyasi',

```

Comment:
Just noticed this looks the same as `icoFile` - is that expected?

Reply:
Oh no, yes it must be changed.